libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -664,7 +664,7 @@ bool AP_Arming::system_checks(bool report)
         }
     }
     if (AP::internalerror().errors() != 0) {
-        check_failed(ARMING_CHECK_NONE, report, "Internal errors detected (0x%x)", AP::internalerror().errors());
+        check_failed(ARMING_CHECK_NONE, report, "Internal errors (0x%x)", AP::internalerror().errors());
```

Comment:
Without the detected just "Internal errors 0x80" would probably be better then using the parens, but thats into the nitpick realm :)